core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -1087,12 +1118,11 @@ object QualificationAppInfo extends Logging {
   }
 
   // Summarize and estimate based on wall clock times
-  def calculateEstimatedInfoSummary(estimatedRatio: Double, sqlDataFrameDuration: Long,
+  def calculateEstimatedInfoSummary(estimatedRatio: Double, sqlDataFrameDurationToUse: Long,
```

Comment:
lets rename some of these variables to inidcate if wall clock time or not sqlDataFrameDurationToUse and appDuration explicilty